datasources-agroal/src/main/java/org/wildfly/extension/datasources/agroal/DataSourceOperations.java

```diff
@@ -39,6 +40,12 @@
 import org.jboss.as.controller.registry.Resource;
 import org.jboss.dmr.ModelNode;
 import org.jboss.msc.service.ServiceName;
+import org.wildfly.common.function.ExceptionSupplier;
+import org.wildfly.security.auth.client.AuthenticationContext;
+import org.wildfly.security.credential.source.CredentialSource;
+
+import java.util.function.Consumer;
+import java.util.function.Supplier;
```

Comment:
Don't change this unless there is some other change needed, but the imports here are out of order.

The jakarta.transaction one was already out of order due to my sed change from javax to jakarta, which took it out of the "java. and javax. before others" category.

Reply:
Fixed